configure.ac

```diff
@@ -1511,20 +1511,28 @@ fi
 
 OPT_DARWINSSL=no
 AC_ARG_WITH(darwinssl,dnl
-AC_HELP_STRING([--with-darwinssl],[enable Apple OS native SSL/TLS])
-AC_HELP_STRING([--without-darwinssl], [disable Apple OS native SSL/TLS]),
+AC_HELP_STRING([--with-darwinssl],[enable use of Secure Transport])
```

Comment:
I think we should keep the original language here & in other places in configure.ac, for two reasons: Schannel is called "Windows native SSL/TLS," and it might not be obvious to everyone that Secure Transport is part of Apple's Security framework.